depthimage_to_navscan_rgbd/launch/example_config.yaml

```diff
@@ -0,0 +1,8 @@
+map_frame: "base_link"
```

Comment:
Move this file to parameter folder

Reply:
Keeping the number of folders down is a good thing, but then rename the folder to example. But in the launch folder shouldn't be any config files.

Reply:
I dont think its that big of a problem but fine